docs/circuits.ipynb

```diff
@@ -773,63 +835,45 @@
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "knjYZkXXAwDO",
+        "id": "Ecsv-L6G0Xdg",
         "colab_type": "text"
       },
       "source": [
-        "### Importing from OpenQASM\n",
-        "\n",
-        "The QASM importer is in an experimental state and currently only supports a subset of the full **OpenQASM spec**. Amongst others, classical control, arbitrary gate definitions, and even some of the gates that don't have a one-to-one representation in Cirq, are not yet supported. The functionality should be sufficient to import interesting quantum circuits. Error handling is very simple - on any lexical or syntactical error, a `QasmException` is raised.\n",
-        "\n",
-        "#### Importing cirq.Circuit from QASM format\n",
+        "## Circuit Optimizers\n",
         "\n",
-        "Requirements: `ply`\n",
+        "Cirq comes with the concept of an optimizer.  Optimizers will pass over a circuit and perform tasks that will modify the circuit in place.  These can be used to transform a circuit in specific ways, such as combining single-qubit gates, commuting Z gates through the circuit, or readying the circuit for certain hardware or gate set configurations.\n",
         "\n",
-        "```\n",
-        "pip install ply==3.4\n",
-        "```"
-      ]
-    },
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "TXFjdV6BAwDP",
-        "colab_type": "text"
-      },
-      "source": [
-        "The following call will create a circuit defined by the input QASM string:"
+        "Optimizers will have a function `optimize_circuit()` that can be used to perform this optimization.  Here is a simple example that removes moments that are empty:"
```

Comment:
nit: change `moments that are empty` to `empty moments`

Reply:
Done.